jena/src/main/scala/org/w3/banana/jena/io/JenaRDFWriter.scala

```diff
@@ -71,4 +71,17 @@ object JenaRDFWriter {
 
   implicit val n3Writer: RDFWriter[Jena, Try, N3] = makeRDFWriter[N3](RDFLanguages.N3)
 
+  implicit val jsonldCompactedWriter: RDFWriter[Jena, Try, JsonLdCompacted] =
```

Comment:
Yes I compared the actual JSON-LD output, and it fits within the definition of compacted as written on the W3C site.